modules/audio_device/audio_device_impl.cc

```diff
@@ -178,7 +178,8 @@ int32_t AudioDeviceModuleImpl::CreatePlatformSpecificObjects() {
       (audio_layer == kPlatformDefaultAudio)) {
     RTC_LOG(INFO) << "Attempting to use the Windows Core Audio APIs...";
     if (AudioDeviceWindowsCore::CoreAudioIsSupported()) {
-      audio_device_.reset(new AudioDeviceWindowsCore());
+      //audio_device_.reset(new AudioDeviceWindowsCore());
+      audio_device_.reset(new AudioDeviceDummy());
```

Comment:
Instead of setting this here, we should define WEBRTC_DUMMY_AUDIO_BUILD to avoid editing the Webrtc codebase.